app/api.py

```diff
@@ -18,24 +19,26 @@
 #MODELs
 
 class User(db.Model):
-	"""
-	User Table Schema
-	"""
-	__tablename__ = "users"
+    """
+    User Table Schema
+    """
+    __tablename__ = "user"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    public_id = db.Column(db.String(50), unique=True)
+    name = db.Column(db.String(50))
+    email = db.Column(db.String(50))
+    password = db.Column(db.String(50))
 
-	id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-	public_id = db.Column(db.String(50), unique=True)
-	name = db.Column(db.String(50))
-	email = db.Column(db.String(50))
-	password = db.Column(db.String(50))
+    events = db.relationship('Event', secondary='reservations',  backref='user', lazy='dynamic')
```

Comment:
I had asked about this in an earlier commit, glad to see it implemented. Squash your commits next time please.